Documentation/Legacy/Index.rst

```diff
@@ -59,19 +59,24 @@ i.e. if you use your own update mechanism.
 
 This feature is already disabled when TYPO3 is installed via Composer.
 
-To disable the core updater, you can set this environment variable::
+To disable the Core updater, you can set this environment variable:
+
+.. code-block:: none
+   :caption: Environment variable
```

Comment:
Would suggesting `.env` imply too much / cause confusion?

Reply:
I think that would only work in docker?

Reply:
The Guis of some webproviders also allow to set this. I would just leave it like that. We have two examples for common cases so this is just the general case.

Reply:
OK.